docs/PacketFence_Installation_Guide.asciidoc

```diff
@@ -4123,9 +4119,7 @@ Next, go in the 'NTLM cache' tab and:
  * Adjust the 'Expiration'
  * Enable 'NTLM cache background job' and/or 'NTLM cache on connection'. In the case of this example, both will be enabled.
 
-image::domain-ntlm-cache-1.png[scaledwidth="100%",alt="Domain NTLM cache"]
-
-image::domain-ntlm-cache-2.png[scaledwidth="100%",alt="Domain NTLM cache"]
+image::domain-ntlm-cache.png[scaledwidth="100%",alt="Domain NTLM cache"]
```

Comment:
- Remove old domain-ntlm-cache-X files if they are not used anymore.
- GitHub report a conflict on domain-ntlm-cache.png that prevent a merge